lib/google-analytics.js

```diff
@@ -52,30 +44,35 @@ module.exports = (function () {
 
     GoogleAnalytics.getCid(function (cid) {
       _.extend(params, { cid: cid }, GoogleAnalytics.defaultParams());
-      GoogleAnalytics.request('https://www.google-analytics.com/collect?' + querystring.stringify(params));
+      this.request('https://www.google-analytics.com/collect?' + querystring.stringify(params));
```

Comment:
`this` is not defined here. You'd need to change the function definition to `(cid) => {`.